test/find-node-version.js

```diff
@@ -59,6 +59,12 @@ describe('find-node-version', function () {
       done();
     });
 
+    it('finds node version from options.nodeVersion', function (done) {
```

Comment:
[nitpick] Change `'finds node version from options.nodeVersion'` to `'sets node version from --node-version'` or something similar